### DISCUSSION TITLE
What is the minimal project needed to run the json sample main?

### DISCUSSION THREAD

**Discussion by mikalziane** [bonede/tree-sitter-ng#29] [Q&A]
Hello there,

In the Getting Started section of the README what do you need to clone in order to run the json sample main method? The whole project is of course too large.

**Comment by bonede** [comment]
Just add the tree sitter and JSON language dependencies to your projects.

**Reply by mikalziane** [reply]
Thank you for your answer.
Let me rephrase my question.
In order to use tree sitter ng, what is the dependency to ng to add?
Your answer suggests that there is none, although looking more carefully it seems that both dependencies are to ng.
I suppose the names of these dependencies may include ng to be more explicit.
I would also suggest to include the imports in the example as I had to struggle to decide what to add.
Finally the announced Java parsing example (instead of Json) would be much appreciated.
Thanks a lot for this work.
